arrow-optics.api

```diff
@@ -229,32 +201,69 @@ public final class arrow/optics/Getter$DefaultImpls {
 	public static fun zip (Larrow/optics/Getter;Larrow/optics/Getter;)Larrow/optics/Getter;
 }
 
-public final class arrow/optics/IdKt {
-	public static final fun toPValue (Larrow/core/Id$Companion;)Larrow/optics/PIso;
-	public static final fun toValue (Larrow/core/Id$Companion;)Larrow/optics/PIso;
+public final class arrow/optics/GetterKt {
```

Comment:
Curious with the new subtyping encoding how the java generation part looks like and if it's friendlier to consume from Javawithout the _Kt files